omnigibson/envs/env_base.py

```diff
@@ -290,6 +305,18 @@ def _load_observation_space(self):
 
         # Also load the task obs space
         obs_space["task"] = self._task.load_observation_space()
+
+        # Also load any external sensors
+        if self._external_sensors is not None:
+            external_obs_space = dict()
+            for sensor_name, sensor in self._external_sensors.items():
+                # Load the sensor observation space
+                external_obs_space[sensor_name] = sensor.load_observation_space()
+                # sensor_obs_space = sensor.load_observation_space()
+                # for obs_modality, obs_modality_space in sensor_obs_space.items():
+                #     external_obs_space[f"{sensor_name}_{obs_modality}"] = obs_modality_space
```

Comment:
Why do we not need to iterate through `obs_modality`? Say there are RGB and depth, they should be two different keys, right?

Reply:
they're nested now! (see comment below)